pedialabsnew/settings_shared.py

```diff
@@ -31,15 +31,12 @@
     'pedialabsnew.main',
     'pedialabsnew.exercises',
     'pedialabsnew.rstplot',
-    'django_markwhat',
```

Comment:
'django_markwhat' should be replaced with: 'markdownify.apps.MarkdownifyConfig'

like this: https://github.com/ccnmtl/worth2/pull/2727/files#diff-c90560fd8ea2506c349b1cda85873461b23c2f4a1f8eb6c804005d3623d8427eR55

Apologies, this isn't documented in our wiki page.


And then, in the template code,
```
{% load markwhat %}
```

will be replaced with: 
```
{% load markdownify %}
```


And any instance of markdown use in this template, defined like, e.g.: `{{ content|markdown }}` will be replaced with `{{ content|markdownify }}`. See: https://github.com/ccnmtl/worth2/pull/2727/files#diff-c90560fd8ea2506c349b1cda85873461b23c2f4a1f8eb6c804005d3623d8427eR55